metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -150,12 +150,22 @@ def get_query_result(self, query: str) -> RowIterator:
     def get_projects(self) -> List[BigqueryProject]:
         with self.report.list_projects:
             try:
-                projects = self.bq_client.list_projects()
-
-                return [
-                    BigqueryProject(id=p.project_id, name=p.friendly_name)
-                    for p in projects
-                ]
+                projects: List[BigqueryProject] = []
+                page_token: Optional[str] = None
+                while True:
+                    projects_iterator = self.bq_client.list_projects(
+                        max_results=5, page_token=page_token
```

Comment:
why is max_results=5?

Reply:
Sorry, this should be 50 as API returns max 50 projects per page. Refer [link](https://cloud.google.com/bigquery/docs/reference/rest/v2/projects/list#query-parameters)